src/Symfony/Component/Messenger/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ CHANGELOG
  * Add `TransportNamesStamp` to change the transport while dispatching a message
  * Add support for rate limited transports by using the RateLimiter component.
  * Deprecate `MessageHandlerInterface` and `MessageSubscriberInterface`, use `#[AsMessageHandler]` instead
+ * Add option `allow_no_senders` to enable throwing when a message doesn't have a sender
```

Comment:
this is for FrameworkBundle, telling about option `framework.messenger.buses.*.default_middleware.allow_no_senders`
here, we should tell about the new argument of SendMessageMiddleware